iroh-net/src/stun.rs

```diff
@@ -508,6 +510,11 @@ mod tests {
         assert_eq!(got_tx, tx);
     }
 
+    #[test]
+    fn test_stun_cookie() {
+        assert_eq!(stun_rs::MAGIC_COOKIE, COOKIE);
+    }
```

Comment:
we can do this in compile time

Reply:
the equality check/conversion is not constant time, so not sure how to do that